src/mscorlib/src/System/ReadOnlySpan.cs

```diff
@@ -328,7 +328,7 @@ public T[] ToArray()
         public static ReadOnlySpan<T> Empty => default(ReadOnlySpan<T>);
     }
 
-    public static class ReadOnlySpanExtensions
+    public static class ReadOnlySpanExtensionMethods
```

Comment:
How about removing the extensions classes from CoreCLR altogether and using the CoreFX implementations (with whichever appropriate refactoring of the classes we decide upon in CoreFX).